package.json

```diff
@@ -0,0 +1,8 @@
+{
+  "devDependencies": {
+    "husky": "^9.1.7"
```

Comment:
Is this actually in use somehow? :)

Reply:
If you're unfamiliar, [Husky](https://www.npmjs.com/package/husky) is a utility package that sets up a git hook in the project to automate tasks like formatting and linting code before commits are pushed.

In our case it lints + formats the code by executing [this lint-staged script](https://github.com/sleeyax/mitmproxy/blob/b159d669c1fe63bf8c09fcbd7eceeb58a2022252/.husky/pre-commit) which in turn executes [the following commands](https://github.com/sleeyax/mitmproxy/blob/7c2f57c351524ea9135013bdb03adc402cd05389/webnext/package.json#L17-L22) in order. 

Totally up for debate if we want to include this or not, it just made my life easier when I worked on this fork initially.

Reply:
Thanks! Somehow missed `.husky` on the branch.

I have opinions on Git pre-commit hooks (which made me build autofix.ci). Strong +1 on having an easy way to do linting/formatting/... locally, but I'll see if Husky is worth it here. :)